examples/metrics/exemplar/src/main/resources/application.yaml

```diff
@@ -22,3 +22,5 @@ server:
   host: 0.0.0.0
 tracing:
   service: "hello-world"
+metrics:
```

Comment:
Is `permit-all` required ? If there is no security, users shouldn't be required to specify `permit-all`.

Reply:
this is intentionally done so you cannot access the endpoint even if you do not have security configured. To protect people who forget they are exposing an endpoint with data that could be missused.

So you have two options:
1.  configure real security and protect the endpoint
2. explicitly mark the endpoint to `permit-all`, so you make an action of affirmation of this problem

Reply:
I am confused. I thought the decision was: if security is not configured for the application then the observability endpoints do not require authorization (and do not require permit-all). If you configure security for the application then the endpoints require authorization -- and you must add permit-all if you do not want the endpoints secured.

That's the behavior we agreed to and that was implemented  in PR #7572.

Reply:
The reason we have common-security is exactly to enforce security even when security is not present, for cases where you could shoot yourself in the leg.
So config, log can NEVER be exposed without authorization, unless you explicitly configure Helidon to allow it